MachObfuscatorTests/DependencyAnalysisTests/ObfuscableFilesFilter_Tests.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ObfuscableFilesFilter.swift
+//  MachObfuscatorTests
+//
+
+import XCTest
+
+private extension URL {
+    init(forFramework framework: String) {
+        self.init(fileURLWithPath: "AppDir/Frameworks/\(framework).framework/\(framework)")
+    }
+}
+
+class ObfuscableFilesFilter_Tests: XCTestCase {
+    func test_shouldObfuscateOnlyApp_whenSkipAllFrameworks() {
+        let withWhitelist = ObfuscableFilesFilter.defaultObfuscableFilesFilter()
+            .and(ObfuscableFilesFilter.skipAllFrameworks())
+            // last condition is added automatically by Obfuscator
+            .and(ObfuscableFilesFilter.onlyFiles(in: URL(fileURLWithPath: "AppDir")))
+
+        // should obfuscate app and nothing else
+        XCTAssert(withWhitelist.isObfuscable(URL(fileURLWithPath: "AppDir/App")))
+
+        XCTAssert(!withWhitelist.isObfuscable(URL(forFramework: "GoodFramework")))
```

Comment:
Why not using `XCTAssertFalse`? This way you could drop the `!` and make the test a little more readable.

Reply:
will change that